src/LiveComponent/tests/Fixture/Kernel.php

```diff
@@ -61,9 +62,16 @@ protected function configureContainer(ContainerBuilder $c, LoaderInterface $load
         // disable logging errors to the console
         $c->register('logger', NullLogger::class);
 
-        $c->register(Component1::class)->setAutoconfigured(true)->setAutowired(true);
-        $c->register(Component2::class)->setAutoconfigured(true)->setAutowired(true);
-        $c->register(Component3::class)->setAutoconfigured(true)->setAutowired(true);
+        $componentA = $c->register(Component1::class)->setAutoconfigured(true)->setAutowired(true);
+        $componentB = $c->register(Component2::class)->setAutoconfigured(true)->setAutowired(true);
+        $componentC = $c->register(Component3::class)->setAutoconfigured(true)->setAutowired(true);
+
+        if (self::VERSION_ID < 50300) {
```

Comment:
Shouldn't this be lower than 8? Since we autoconfigure with attributes now.

Reply:
I'm checking the Symfony version here. Auto configuration with attributes is only available in Symfony 5.3+.

Reply:
Ahh sorry. Cool 👍